design-tokens/tokens/semantic/dimension.small.json

```diff
@@ -298,7 +298,7 @@
       "xxsmall": {
         "$type": "number",
         "$value": "{static.container.xxsmall}",
-        "$description": "Container tokens define the minimum/maximum width and height of containers or layout elements. For example, box, card and grids.",
+        "$description": "Use to define the minimum/maximum width and height of containers or layout elements. For example, box, card and grids.",
```

Comment:
FYI -- No changes in this PR affected this, but rather the descriptions between the "medium" and "small" mode in our Github files had gotten out of sync. When pulling from Figma in this PR, this will align the descriptions.